examples/sentencepiece/README.md

```diff
@@ -0,0 +1,13 @@
+# This is an example to create a dataset with SentencePiece binding. 
```

Comment:
nit: title should probably just be "# Create a dataset with SentencePiece binding" since the text already refers to the content as an example, and its in the example folder :)